assignments/function-conversion.js

```diff
@@ -4,22 +4,33 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+const myFunction = () => { "Function was invoked!" };
```

Comment:
Love it! .. 
You COULD drop curly brackets if you'd like! :)
const myFunction = () => "Function was invoked!";